rest-api/offline/public_metrics_export.py

```diff
@@ -0,0 +1,145 @@
+from sqlalchemy import text
+from dao import database_factory
+from dao.database_utils import replace_years_old
+from participant_enums import EnrollmentStatus, OrderStatus, PhysicalMeasurementsStatus
+from participant_enums import Race, QuestionnaireStatus
+
+# TODO(calbach): consider whether we need to filter down to consented
```

Comment:
All of these use participant_summary, and thus only operate on consented participants. (Which I think is totally fine and appropriate.)